src/main/resources/frontend/src/utils/utils.ts

```diff
@@ -19,8 +19,12 @@ export const commitVerificationFailed = (submission: Submission) => {
   return submission.verifiedStatus.toString() === VerifiedStatus[VerifiedStatus.Unapproved];
 }
 
+export const isNeverTimestamp = (timestampString: Date | string): boolean => {
+  return timestampString === "+1000000000-12-31T23:59:59.999999999Z" // The Java Instant.MAX value
```

Comment:
the way things are structured for both system shutdown and banner-message expiration, that random(ish) string is Java Instant.MAX. We're forced to put a timestamp in for those config values, it won't accept null, so we have to just accept using Instant.MAX.

The problem you point out those is a valid one. I can go change it so that the server side just sends a string "never" in the config json if the timestamp in the server is Instant.MAX. That way the front end doesn't care about what that value is.